test/Microsoft.AspNet.Routing.Tests/Template/TemplateRouteTest.cs

```diff
@@ -1070,7 +994,8 @@ public void GetVirtualPath_Sends_ProvidedValues()
                 .Returns<string>(null);
 
             var route = CreateRoute(target.Object, "{controller}/{action}");
-            var context = CreateVirtualPathContext(new { action = "Store" }, new { Controller = "Home", action = "Blog" });
+            var context = CreateVirtualPathContext(
+                new { action = "Store" }, new { Controller = "Home", action = "Blog" });
```

Comment:
if you're putting a new line in a method call, put each parameter on its own line